pkg/project/project.go

```diff
@@ -358,6 +358,36 @@ func FromExactPath(path string) (*Project, error) {
 	return project, nil
 }
 
+// LocalProjectDoesNotExist is an error returned when a requested project is not checked out locally.
+type LocalProjectDoesNotExist struct{ error }
+
+// IsLocalProjectDoesNotExist checks if the error is a LocalProjectDoesNotExist.
+func IsLocalProjectDoesNotExist(err error) bool {
+	return errs.Matches(err, &LocalProjectDoesNotExist{})
```

Comment:
Personally I would prefer using this code directly instead of a helper function. It's concise and easy to understand, the helper just creates friction.

Reply:
I was following this convention: https://github.com/ActiveState/cli/blob/e94d0b485e7775e975efa2ce6a27b15be86a5094/pkg/platform/runtime/ARCHITECTURE.md